dataset/dictionary/dialogue.json5

```diff
@@ -314,6 +317,7 @@
   {
     en: "Have a safe trip",
     ja: "行ってらっしゃい",
+    // TODO add zhCN
```

Comment:
This Wriothesley's dialogue can be found in the special program. I guess the same video was broadcasted on Bilibili.
 https://youtu.be/NUBIYZ_KQFc?si=zT6-j1PeGFNjnEVW&t=681

Reply:
I checked it out on Bilibili.

```
zhCN: "一路顺风",
```

Reply:
@Bill-Haku Thanks!

@SleepyAsh0191 Can you add this?

Reply:
done

Reply:
@SleepyAsh0191 Thanks, merged!